chowpronto_frontend/src/patron/pages/confirmOrder/components/Login.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "styled-components";
+import ChowButton from "../../../../shared_components/ChowButton";
+import { StyledBoxContainer } from "../../../../shared_components/BoxContainer";
+
+export default function Login() {
+  return (
+    <Divider>
+      <h3>Already Registered?</h3>
+      <ChowButton primary elevated title="login" />
+    </Divider>
+  );
+}
```

Comment:
Do we only want to highlight the box that contains the Button?  Users might be confused if the whole box lights up and think they need to press it rather than the Button.

Reply:
Update:
We can also change this in a later PR.